shapeout/gui/plot_scatter.py

```diff
@@ -204,13 +215,13 @@ def set_scatter_data(plot, mm):
         positions = None
     else:
         print("...Downsampled from {} to {} in {:.2f}s".format(lx, x.shape[0], time.time()-a))
-        positions = np.vstack([x.ravel(), y.ravel()])
-
+        #Apply log-scaling before density is computed     
```

Comment:
please remove empty spaces from line ending and start comments with a hash and a space, i.e. ``# Apply ...``)